frontend/app/views/offer/offersandcomps.scala.html

```diff
@@ -0,0 +1,50 @@
+@(membersOnlyContent: Seq[model.MembersOnlyContent], noResultText: String)
+
+@main("Offers & Competitions") {
+    <main role="main" class="l-constrained">
+        <section class="listing-header">
+            <h1 class="listing-headline">Offers and competitions</h1>
+        </section>
+        <div class="event-blocks">
+            <section class="listing listing--lead listing--padded">
+                <div class="listing__lead-in">
+                    <h3 class="listing__title h-intro">most recent</h3>
+                </div>
+                <div class="listing__content">
+                @if(membersOnlyContent.isEmpty) {
+                    <div class="listing__empty">@noResultText</div>
+                } else {
+                    <ul class="grid grid--bordered grid--3up">
+                    @for(memberContent <- membersOnlyContent) {
+                        <li class="grid__item">
+                            <a href="@memberContent.content.webUrl" class="event-item event-item--small" itemscope itemtype="http://schema.org/Event">
+                                <meta itemprop="url" content="@memberContent.content.webUrl">
+                                <div class="event-item__media" itemprop="image" content="@memberContent.mainImg">
+                                    <img class="responsive-img"
+                                         src="@memberContent.mainImg"
+                                         srcset="@memberContent.srcset"
+                                         sizes="(min-width: 739px) 33.3vw, (min-width: 479px) 50vw, 100vw"
+                                         alt=""/>
+                                </div>
+                                <div class="event-item__content">
+                                    <div class="event-item__meta">
+                                        <h4 class="event-item__title" itemprop="name">
+                                            @fragments.inlineIcon("outbound-link", Seq("icon-inline--brand-dark"))
+                                            @if(memberContent.tagTitle) {
+                                                <span class="tone-outbound-link">@memberContent.tagTitle</span>
```

Comment:
Not sure if this should be using tone colours. Might be worth creating a new `.link-outbound` component which wraps all this up.